### ISSUE TITLE
Admin popup with error connection-failedhostname-mismatch

### ISSUE THREAD

**Issue by marcopolo88** [raspgot/AjaxForm-PHPMailer-reCAPTCHA#2]
on form submission:
Admin popup with errror message: connection-failedhostname-mismatch
of course the site keys with google recaptcha v3 were registered with correct domain both with and w/o www
sometimes in v2 there were request problems with ssl protected forms, cannot believe that is still problematic?
have you got an idea @raspgot raspgot.fr Gauthier?
Merci

**Comment by raspgot** [comment]
Hello @marcopolo88 
![Screenshot_20200811-094805~2.png](https://user-images.githubusercontent.com/16710364/89872014-9b0ce380-dbb8-11ea-9257-206e02442da2.png)
I don't see the error code reference in doc, what's the domain you have registered in Google admin dashboard ?

**Comment by adriy-be** [comment]
Hi, I have got the same error `connection-failed hostname-mismatch` after some time of search and debug. The `hostname-mismatch` is just a bug due to `connection-failed`.

On my side, the error `connection-failed` is due to the missing package `php-openssl` !
I have added this package, and now it's working fine.

I have also modified AjaxForm.js line ~49 to don't reset the form on error.
```
if(txt.error === false)
    form.reset();
```

Thanks
